tests/depth_of_investigation_tests/sensitivity_cutoff_test.py

```diff
@@ -0,0 +1,118 @@
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+#  Copyright (c) 2024 Mira Geoscience Ltd.
+#  All rights reserved.
+#
+#  This file is part of simpeg-drivers.
+#
+#  The software and information contained herein are proprietary to, and
+#  comprise valuable trade secrets of, Mira Geoscience, which
+#  intend to preserve as trade secrets such software and information.
+#  This software is furnished pursuant to a written license agreement and
+#  may be used, copied, transmitted, and stored only in accordance with
+#  the terms of such license and with the inclusion of the above copyright
+#  notice.  This software and information or any other copies thereof may
+#  not be provided or otherwise made available to any other person.
+#
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+import numpy as np
+from geoh5py import Workspace
+from geoh5py.objects import Points
+from octree_creation_app.driver import OctreeDriver
+from octree_creation_app.params import OctreeParams
+
+from simpeg_drivers.depth_of_investigation.sensitivity_cutoff.driver import (
+    SensitivityCutoffDriver,
+    scale_sensitivity,
+)
+from simpeg_drivers.depth_of_investigation.sensitivity_cutoff.params import (
+    SensitivityCutoffParams,
+)
+
+
+def generate_sensitivity(workspace: Workspace, depth_scaling=False):
+    x = np.linspace(-500, 500, 21)
+    y = np.array([-200, -100, 0, 100, 200])
+    x_grid, y_grid = np.meshgrid(x, y)
+    vertices = np.c_[
+        x_grid.flatten(), y_grid.flatten(), np.zeros_like(x_grid.flatten())
+    ]
+    survey = Points.create(workspace, name="survey", vertices=vertices)
+    refinements = {
+        "Refinement A object": survey,
+        "Refinement A levels": "4, 4, 4",
+        "Refinement A horizon": False,
+    }
+    octree_params = OctreeParams(
+        geoh5=workspace,
+        objects=survey,
+        u_cell_size=25,
+        v_cell_size=25,
+        w_cell_size=25,
+        padding_distance=1000,
+        depth_core=500,
+        minimum_level=4,
+        diagonal_balance=True,
+        **refinements,
+    )
+    mesh = OctreeDriver.octree_from_params(octree_params)
+
+    values = (
+        mesh.octree_cells["NCells"]
+        * mesh.u_cell_size
+        * mesh.octree_cells["NCells"]
+        * mesh.v_cell_size
+        * mesh.octree_cells["NCells"]
+        * mesh.w_cell_size
```

Comment:
```suggestion
        mesh.octree_cells["NCells"]**3,
        * mesh.u_cell_size
        * mesh.v_cell_size
        * mesh.w_cell_size
```

Reply:
Yep, I noticed that on your earlier comment and updated in the geoh5py PR

Reply:
You can use geoh5py attribute now.

Reply:
done.